modules/migrating-to-virt.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/getting_started/virt-getting-started.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="migrating-to-virt_{context}"]
+= Migrating to {VirtProductName}
+
+To migrate virtual machines from an external provider such as VMware vSphere, Red Hat OpenStack, Red Hat Virtualization, or another {VirtProductName} cluster, use the Migration Toolkit for Virtualization (MTV). You can also migrate Open Virtual Appliance (OVA) files created by VMware vSphere.
+
+[NOTE]
+====
+Migration Toolkit for Virtualization is not part of {VirtProductName} and requires separate installation. Because of that, all links in this procedure lead outside of {VirtProductName} documentation.
+====
+
+.Prerequisites
+* The Migration Toolkit for Virtualization Operator link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/installing-the-operator_mtv[is installed].
+
+.Procedure
+. link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console_mtv#adding-source-providers[Configure the source provider].
++
+Select VMware vSphere, Red Hat OpenStack, Red Hat Virtualization, OVA files created by VMware vSphere, or another {VirtProductName} cluster as the source provider. Depending on the selection, enter required information such as the URL of the API endpoint or credentials needed to access the virtual machines.
```

Comment:
This might be my personal opinion, but I don't think we need to mention the list of source providers here. Suggestion for minimalism:

```suggestion
Select the source provider and enter any required information, such as the URL of the API endpoint or credentials needed to access the virtual machines.
```